examples/vanilla/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "remeda-example-vanilla",
+  "version": "1.0.0",
+  "main": "index.html",
+  "scripts": {
+    "start": "parcel index.html --open",
+    "build": "parcel build index.html"
+  },
+  "dependencies": {
+    "parcel-bundler": "^1.6.1",
+    "remeda": "^1.0.0"
```

Comment:
Would this example use remeda build from pr?

Reply:
yes, the remeda dependency is replaced with a preview build from this PR.
take a look at the above link: https://codesandbox.io/s/remeda-example-vanilla-bhnq8p

it depends on:

`"remeda": "https://pkg.csb.dev/remeda/remeda/commit/1a558604/remeda"`

which is the last commit of this PR. it will always be re-built with every push.

Reply:
Magic :)